CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Added 
+* Add k8s 1.27 support by @csongnr in [#845](https://github.com/newrelic/nri-kubernetes/pull/845)
```

Comment:
Can you move this under `### enhancement` like this:

```
### enhancement
- Add k8s 1.27 support by @csongnr in [#845](https://github.com/newrelic/nri-kubernetes/pull/845)
```
We implemented a changelog requirement in order to use a release toolkit to automate releases and this means we have to follow a format for the changelog. 

We added some notes about that here: https://github.com/newrelic/nri-kubernetes/blob/main/CONTRIBUTING.md#pull-requests

So `### Added` is not an accepted header anymore.

Reply:
ah, got it. yeah i saw the requirement was an L3 header which was why i added it, didn't realize thats what the enhancements header was supposed to be for :D